### PR TITLE
Feature/basic est update

### PR DESCRIPTION
**Pull request by andrjohns**
Updates the models in ```basic_estimators``` with modern Stan syntax (vectorisation, ```_lupdf```, etc)

New syntax parses in Stanc3 (RStan 2.26) with no warnings or errors.

### CODE REVIEW

File: basic_estimators/normal_censored.stan [hunk 1]
```diff
@@ -8,10 +8,10 @@ parameters {
   real mu;
 }
 model {
+  // Vectorization with truncation is not yet supported
```
Comment (rok-cesnovar) [inline basic_estimators/normal_censored.stan:11]: Probably no need for this comment.

File: basic_estimators/normal_truncated.stan [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-  real<upper=U> y[N];
+  vector<upper=U>[N] y;
 }
 parameters {
   real mu;
-  real<lower=0,upper=2> sigma;
+  real<lower=0, upper=2> sigma;
 }
 model {
+  // Truncation not yet available with vectorised distributions
```
Comment (rok-cesnovar) [inline basic_estimators/normal_truncated.stan:11]: Same as above.

**Review by rok-cesnovar — CHANGES_REQUESTED** [review]
Thanks Andrew! Looks great. 

We probably dont need the comment on vectorization as it could just confuse the user.

**Comment by andrjohns** [comment]
Good call! Have updated

**Review by rok-cesnovar — APPROVED** [review]
:tada:

**Comment by rok-cesnovar** [comment]
Thanks Andrew. I gave you write permission to the repo so you can do the branches here, if that helps you in any way.

**Pull request merged**